testnet_deployed.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
same with this file? 
are these temporary files?

Reply:
nope, not temporary, they will be generated and filled from the node